src/main/scala/housekeeper/Lambda.scala

```diff
@@ -15,16 +16,23 @@ object Lambda extends Logging {
 
     val bouncedAddresses = bounce.bouncedEmailAddresses.toSeq.sorted
 
+    val senderOfBouncedEmail = bounceNotification.mail.source
+    val bounceSummary = s"$senderOfBouncedEmail sent a ${bounce.bounceType} bounced message to ${bouncedAddresses.mkString(",")}"
     logger.info(Map(
       "bounce.type" -> bounce.bounceType,
       "bounce.subtype" -> bounce.bounceSubType,
       "bounce.permanent" -> bounce.isPermanent,
-      "bounce.mail.source" -> bounceNotification.mail.source,
+      "bounce.mail.source" -> senderOfBouncedEmail,
       "bounce.bouncedEmailAddresses" -> bouncedAddresses.asJava
-    ), s"${bounceNotification.mail.source} sent a ${bounce.bounceType} bounced message to ${bouncedAddresses.mkString(",")}")
+    ), bounceSummary)
 
     if (bounce.isPermanent) {
       bouncedAddresses.foreach(alertDeletion.deleteAllAlertsForEmailAddress)
+      if (bounce.isOnSuppressionList) {
```

Comment:
From https://docs.aws.amazon.com/ses/latest/DeveloperGuide/remove-from-suppression-list.html :

> Amazon SES maintains a global suppression list. This list contains recipient email addresses that have recently caused a hard bounce for any Amazon SES customer. If you try to send an email through Amazon SES to an address that's on the global suppression list, the call to Amazon SES succeeds, but Amazon SES treats the email as a hard bounce instead of attempting to send it. Like any hard bounce, global suppression list bounces count towards your sending quota and your bounce rate.

It's legitimate that we sometimes get 'Permanent' bounces (ie when Guardian staff leave the company, and their email account is shutdown) - but if we keep sending email to that address, and it keeps bouncing, Amazon will eventually put it on suppression list. In order to minimise noise on the topic, I've decided to only send the SNS notification when things are _that_ bad - not in the acceptable case where a colleague has simply left the Guardian.